tests/dummy/Queries/PaginatedMoviesFrom2014Query.php

```diff
@@ -3,14 +3,23 @@
 namespace ElasticSearcher\Dummy\Queries;
 
 use ElasticSearcher\Abstracts\AbstractQuery;
+use ElasticSearcher\ElasticSearcher;
 use ElasticSearcher\Fragments\Traits\PaginatedTrait;
 
 class PaginatedMoviesFrom2014Query extends AbstractQuery
 {
 	use PaginatedTrait;
 
-	public function setup()
+	private $pageNumber;
+
+	public function __construct(\ElasticSearcher\ElasticSearcher $searcher, $page = 3)
+  {
+    parent::__construct($searcher);
+    $this->pageNumber = $page;
+  }
+
+  public function setup()
 	{
-		$this->paginate(3, 10);
+		$this->paginate($this->pageNumber, 10);
```

Comment:
We have `addData` for these kind of things, no need to pass it via the constructor.